services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -428,15 +435,72 @@ async def pull_input_ports(
     return transferred_bytes
 
 
+@containers_router.patch(
+    "/containers/directory-watcher",
+    summary="Enable/disable directory-watcher event propagation",
+    response_class=Response,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def disable_directory_watcher(
+    patch_directory_watcher_item: PatchDirectoryWatcherItem,
+    app: FastAPI = Depends(get_application),
+) -> None:
+    if patch_directory_watcher_item.is_enabled:
+        directory_watcher.enable_directory_watcher(app)
+    else:
+        directory_watcher.disable_directory_watcher(app)
+
+
+@containers_router.post(
+    "/containers/ports/outputs/dirs",
+    summary=(
+        "Creates the output directories declared by the docker images's labels. "
+        "Since the dynamic-sidecar cannot retrieve the labels "
+        "from the image, these must be provided by the "
+        "director-v2. This API is called after the outputs directory was pulled"
+    ),
+    response_class=Response,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def create_output_dirs(request_mode: CreateDirsRequestItem) -> None:
+    mounted_volumes: MountedVolumes = get_mounted_volumes()
+    outputs_path = mounted_volumes.disk_outputs_path
+    for port_key, service_output in request_mode.outputs_labels.items():
+        if is_file_type(service_output.property_type):
+            dir_to_create = outputs_path / port_key
+            dir_to_create.mkdir(parents=True, exist_ok=True)
+
+
+@containers_router.post(
+    "/containers/ports/outputs:pull",
+    summary="Pull input ports data",
+    response_model=None,
```

Comment:
``response_model=None,`` is the default, no need to add it

Reply:
but you did not change it!?

Reply:
it's changed now.